conformance/results/kong-kubernetes-ingress-controller.yaml

```
@@ -0,0 +1,55 @@
+typemeta:
+  kind: ""
+  apiversion: ""
+implementation:
+  organization: kong
+  project: kubernetes-ingress-controller
+  url: github.com/kong/kubernetes-ingress-controller
+  version: "2.10"
+  contact:
+  - '@mlavacca'
+  - '@shaneutt'
+date: "2023-07-17T14:54:07+02:00"
+gatewayapiversion: TODO
+profilereports:
+- name: HTTP
+  core:
+    result: partial
+    summary: |
+      'Very first conformance profile.
+      Conformance tests have been run with https://github.com/kubernetes-sigs/gateway-api/commit/7ded95641899d6fb74e60d4793439683430325e7.
+      Full conformance with Gateway-API v0.7.1 (all 23 tests pass).
+      The 6 skipped tests include cases unreleased and pending release in Gateway-API v0.8.'
+    statistics:
+      passed: 23
+      skipped: 6
+      failed: 0
+    skippedtests:
```

Suggestion applies throughout - why not use came case like other k8s APIs here?
```suggestion
    skippedTests:
```

---

There was a bug in the CLI. I've fixed and created a PR [here](https://github.com/kubernetes-sigs/gateway-api/pull/2196).